src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,47 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  position: absolute;
+  width: ($size + $padd) * 7;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+
+  &__day {
+    background-color: #eee;
+    width: $size;
+    height: $size;
+    margin-top: $padd;
+    margin-right: $padd;
```

Comment:
try to use gap for the parent's block

Task
Use flex with 1px gap and limit its width to exactly 7 columns + 10px paddings